opennms-webapp/src/main/java/org/opennms/web/filter/FilterUtil.java

```diff
@@ -55,10 +57,18 @@ public static String toFilterURL(List<String> filters) {
 
     public static String[] parse(String filterString) {
         String decodedString = URLDecoder.decode(filterString, StandardCharsets.UTF_8);
-        String[] filterParameter = (decodedString == null) ? null : Arrays.stream(decodedString.split("&amp;"))
-                .filter(fp -> fp.startsWith("filter="))
+        return Arrays.stream(getFilterParameters(decodedString))
                 .map(fp -> fp.replace("filter=", ""))
```

Comment:
There may be non-filter parameters, right? Do we need a .filter(fp->fp.startsWith("filter=")) first?

Reply:
Yes, there may be other parameters, so adding that would make sense. 

On the other hand, this array eventually makes its way to `EventUtil:getFilterList()` where any non-filter parameters are ignored, so they don't cause any issues. And this is already in `release-30.x`, so modifying it would create (more) merge conflicts for Ben/Morteza to sort through. 

I'm not sure if it's worth adding. Thoughts?

Reply:
Ok